core/src/main/java/com/datastax/oss/driver/api/core/metadata/TokenMap.java

```diff
@@ -99,15 +108,39 @@
 
   /** The replicas for a given partition key in the given keyspace. */
   @NonNull
-  Set<Node> getReplicas(@NonNull CqlIdentifier keyspace, @NonNull ByteBuffer partitionKey);
+  Set<Node> getReplicas(
+      @NonNull CqlIdentifier keyspace,
+      @Nullable Partitioner partitioner,
+      @NonNull ByteBuffer partitionKey);
+
+  /**
+   * Shortcut for {@link #getReplicas(CqlIdentifier, Partitioner, ByteBuffer) getReplicas(keyspace,
+   * null, partitionKey)}.
+   */
+  @NonNull
+  default Set<Node> getReplicas(@NonNull CqlIdentifier keyspace, @NonNull ByteBuffer partitionKey) {
+    return getReplicas(keyspace, null, partitionKey);
+  }
+
+  /**
+   * Shortcut for {@link #getReplicas(CqlIdentifier, Partitioner, ByteBuffer)
+   * getReplicas(CqlIdentifier.fromCql(keyspaceName), partitioner, partitionKey)}.
+   */
+  @NonNull
+  default Set<Node> getReplicas(
+      @NonNull String keyspaceName,
+      @Nullable Partitioner partitioner,
+      @NonNull ByteBuffer partitionKey) {
+    return getReplicas(CqlIdentifier.fromCql(keyspaceName), partitioner, partitionKey);
+  }
```

Comment:
since we've got those shortcuts are really all those changes in the tests necessary?

Reply:
Yes, I think so. I think that those tests directly call `getReplicas(keyspace, partitioner, partitionKey)` with `partitioner = null`, so mocks had to be updated. Without those shortcuts even more places would have to be fixed.